dist/dropdown/ds6/dropdown.css

```diff
@@ -82,13 +93,13 @@ div.menu__item[role^="menuitem"]:hover,
 div.listbox__option[role^="option"]:hover,
 a.fake-menu__item:hover,
 button.fake-menu__item:hover {
-  background-color: #f5f5f5;
+  background-color: #e5e5e5;
 }
 div.menu__item[role^="menuitem"]:focus,
 div.listbox__option[role^="option"]:focus,
 a.fake-menu__item:focus,
 button.fake-menu__item:focus {
-  outline: 1px solid #c7c7c7;
+  outline: none;
```

Comment:
Csshint is throwing a warning here:

```
Outlines shouldn't be hidden unless other visual changes are made.
div.menu__item[role^="menuitem"]:focus
```

I guess we can overrule this line, as we are making other visual changes.

Reply:
How do we overrule it when comments are stripped out?